tools/vipps-partner-api-postman-environment.json

```diff
@@ -4,23 +4,23 @@
 	"values": [
 		{
 			"key": "client_id",
-			"value": "<Use the partner keys you have received>",
+			"value": "<Go To Vipps Portal->Select the Developer tab->Select Test or Production Keys->Select 'Show keys'->copy the Client Id here>",
 			"type": "default",
 			"enabled": true
 		},
 		{
 			"key": "client_secret",
-			"value": "<Use the partner keys you have received>",
+			"value": "<Go To Vipps Portal->Select the Developer tab->Select Test or Production Keys->Select 'Show keys'->copy the Client secret here>",
 			"enabled": true
 		},
 		{
 			"key": "Ocp-Apim-Subscription-Key",
-			"value": "<Use the partner keys you have received>",
+			"value": "<Go To Vipps Portal->Select the Developer tab->Select Test or Production Keys->Select 'Show keys'->copy the Vipps-Subscription-Key>",
 			"enabled": true
```

Comment:
Partners currently have no way of getting their keys from the portal, so we have to provide them to the partner.

Reply:
I will revert back to the previous text. :-)

Reply:
Seems like you didn't revert client_secret and client_id (they are also part of what we call partnerKeys, what we call partnerKeys is all the normal keys, just for partners)

Reply:
Oops! I didn't catch that. I've updated it now.